src/ert/scheduler/lsf_driver.py

```diff
@@ -82,18 +97,26 @@ class RunningJob(BaseModel):
 JOB_ALREADY_FINISHED_BKILL_MSG = "Job has already finished"
 
 
-class _Stat(BaseModel):
-    jobs: Mapping[str, AnyJob]
+def _create_job_class(job_state: JobState) -> AnyJob:
```

Comment:
Since this is only used inside _parse_job_dict, it can probably be inlined there?